api/registrations/views.py

```diff
@@ -155,6 +156,16 @@ class RegistrationList(JSONAPIBaseView, generics.ListAPIView, NodesFilterMixin):
     ordering = ('-date_modified',)
     model_class = Registration
 
+    # overrides BulkUpdateJSONAPIView
+    def get_serializer_class(self):
+        """
+        Use RegistrationDetailSerializer which requires 'id'
+        """
+        if self.request.method in ('PUT', 'PATCH'):
```

Comment:
This is fine for now, but maybe change to `self.request.method not in permissions.SAFE_METHODS` in the future.

Reply:
Good to know :+1:. Was basically copypasta from nodelist https://github.com/CenterForOpenScience/osf.io/blob/develop/api/nodes/views.py#L307

Reply:
I don’t think safe_methods is accurate in this case. Post, get, or delete would go to one branch, and put and patch would go to another.